transformer/task.py

```diff
@@ -185,53 +223,114 @@ def replace_url(self, url: str):
         return self._replace(locust_request=new_locust_request)
 
 
-def _indent(input_string: str, requested_indentation: int) -> str:
```

Comment:
Can you please clarify for me why we don't need this anymore?

Reply:
There is a low-level explanation (PyCharm and grep detect that the function is never used) and a high-level explanation (with the removal of `Task.as_locust_action`, all code generation is now handled by `transformer.python` which implements indentation differently, see [Line](https://github.com/zalando-incubator/Transformer/blob/c96ff1bed109f114f7c69143c85ee362e5f518d1/transformer/python.py#L19-L34)).